src/workspaces/dashboard/WorkspaceTags.ts

```diff
@@ -90,12 +90,12 @@ export const WorkspaceTags = (props: WorkspaceTagsProps): ReactNode => {
         ]),
         canEdit &&
           div({ style: { marginBottom: '0.5rem' } }, [
-            h(WorkspaceTagSelect, {
+            h(WorkspaceTagSelect<false>, {
               menuShouldScrollIntoView: false,
               value: null,
               placeholder: 'Add a tag',
               'aria-label': 'Add a tag',
-              onChange: ({ value }) => addTag(value),
+              onChange: (opt) => addTag(opt?.value),
```

Comment:
I'm actually not 100% sure why the original line fails to typecheck (`value` is a property of the argument), but the update also makes it more clear that the argument to `onChange`  corresponds to the `Option` type used in the `react-select` components.